### ISSUE TITLE
Disappearing numbers/disappearing functionality

### ISSUE THREAD

**Issue by Paws-Deep** [Anarios/return-youtube-dislike#763]
### Browser

Firefox, Chrome

### Browser Version

Firefox 105.0 (64-bit), Chrome 105.0.5195.127 (Official Build) (64-bit)

### Extension or Userscript?

Extension

### Extension/Userscript Version

3.0.0.6

### Video link where you see the problem

any youtube video

### What happened?

1. When pressing "like" button, the number of likes disappears.
2. Or sometimes a page with a video loads and there are just icons for "like" and "dislike", no numbers at all.
3. Or it only shows the number of "likes" and the field for "dislikes" count is empty, but the number of dislikes shows up after pressing "dislike" (which functions as a reverse to the first point above: likes disappear after voting, dislikes only appear after voting).
![image](https://user-images.githubusercontent.com/74794023/191723378-a29110f7-cc9e-4f8e-9341-634b0075643d.png)
![image](https://user-images.githubusercontent.com/74794023/191723655-8b7b82dc-dfb0-4424-873e-626ffc3cd406.png)

### How to reproduce/recreate?

The issue started to occur since a week or two ago, everything worked as intended in August.
I tested both of my browsers (Chrome and Firefox) + in private windows with all other extensions disabled because I suspected a conflict with some different extension might be the culprit.
Nope. The issue persists even if Return dislikes is the only extension. However, it seems I pinned down the exact setting which causes those problems. Which is:
![image](https://user-images.githubusercontent.com/74794023/191725221-882a244f-61bb-4e28-991a-134db1c3cc04.png)
After turning off that "re-format" option, likes/dislikes seemingly start to function ok again.

### Will you be available for follow-up questions to help developers diagnose & fix the issue?

Yes

**Comment by einarenrique** [comment]
I have the exact same problem in Firefox 105.0 in macOS Monterey

**Comment by Musilix** [comment]
I'm having the same issue in Chrome 105.0, but only for the dislikes. On top of that, there's this weird behavior where the dislikes don't show for every OTHER video I click. But they will always show for a video I navigate directly to with a url. And to add on to it all, they will start to show after I click the actual dislike button, just like you. Strange.

**Comment by Paws-Deep** [comment]
> I'm having the same issue in Chrome 105.0, but only for the dislikes. On top of that, there's this weird behavior where the dislikes don't show for every OTHER video I click. But they will always show for a video I navigate directly to with a url. And to add on to it all, they will start to show after I click the actual dislike button, just like you. Strange.

Yep, try turning off that re-format setting if you have it on, or test if maybe some other Return dislikes setting causes this issue for you. 
I just noticed that dunkey's video in my screenshot received 170 809 170 000 likes according to re-format setting. 170 billion likes, no yt video ever received anything close to that XD It must be aliens' conspiracy. Or something's wrong with the count in re-format option, hence all those bugs/glitches)

**Comment by Musilix** [comment]
I noticed for my own problem that the latest build on the main branch seems to fix my problem when I run it locally. More specifically, it seems that [this commit](https://github.com/Anarios/return-youtube-dislike/commit/de1831f2c74032f8ccfe42a8eb174088363e011f) fixed my issues with dislikes not showing. I guess it could be that the latest build of the main branch hasn't been deployed though? I'm not sure.

**Comment by tyharg** [comment]
Changing the number format works for most videos for me (nice find @Paws-Deep), on occasion I have to either click thumbs up or thumbs down for the numbers to show.   Thanks to everyone contributing to this!

**Comment by Mrcubix** [comment]
![image](https://user-images.githubusercontent.com/39861216/192142947-ba3a96a5-e6a9-403c-9b5a-bd9266a20868.png)
